### PR TITLE
Objects moved to src

### PR DESCRIPTION
**Pull request by zeroadam**
BaseObject, Conversation, and Item moved to Friendica\Core namespace.

Related to #3878

### CODE REVIEW

**Comment by annando** [comment]
Great, so we get rid of the "objects" folder!

It is too late for me to have a deep look at the code, but in a quick view it looked good. I guess @MrPetovan will have a deeper look.

**Comment by zeroadam** [comment]
I would've moved TemplateEngine as well, but wasn't sure where it should go. That is the last file in the object folder!

**Comment by annando** [comment]
It belongs to ```include/friendica_smarty.php```. Do we have a namespace for frontend stuff?

File: src/Core/BaseObject.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * @file src/Core/BaseObject.php
+ */
+namespace Friendica\Core;
+
+if (class_exists('BaseObject')) {
```
Comment (rabuzarus) [inline src/Core/BaseObject.php]: @annando 
do we still need this this condition?

Comment (MrPetovan) [reply src/Core/BaseObject.php]: Nope!

Comment (zeroadam) [reply src/Core/BaseObject.php]: done

Comment (rabuzarus) [reply src/Core/BaseObject.php]: ok then in `src /Core/Item.php` is another that can be removed

File: src/Core/Conversation.php [hunk 2]
```diff
@@ -1,51 +1,62 @@
 <?php
-if(class_exists('Conversation'))
+/**
+ * @file src/Core/Conversation.php
+ */
+namespace Friendica\Core;
+
+if (class_exists('Conversation')) {
```
Comment (MrPetovan) [inline src/Core/Conversation.php]: Please remove this condition as well.

File: src/Core/Item.php [hunk 3]
```diff
@@ -1,20 +1,28 @@
 <?php
-if(class_exists('Item'))
+/**
+ * @file src/Core/Item.php
+ */
+namespace Friendica\Core;
+
+if (class_exists('Item')) {
```
Comment (MrPetovan) [inline src/Core/Item.php]: And this condition too!

File: src/Core/BaseObject.php (hunk 1 above)
Comment (zeroadam) [reply src/Core/BaseObject.php]: and Core/Conversation.php too?

**Comment by MrPetovan** [comment]
We don’t have a namespace for frontend stuff. `Render`? ˋDisplay`?

**Comment by rabuzarus** [comment]
Since we have module with the name `display` it could lead to confusion if we would have a namespace `display` for frontend stuff

File: src/Core/Item.php (hunk 3 above)
Comment (zeroadam) [reply src/Core/Item.php]: done

File: src/Core/Conversation.php (hunk 2 above)
Comment (zeroadam) [reply src/Core/Conversation.php]: done

**Comment by MrPetovan** [comment]
`Module` or `Controller`.

**Pull request merged**